src/Microsoft.DotNet.Darc/src/Darc/Operations/AddBuildToChannelOperation.cs

```diff
@@ -106,6 +107,16 @@ private async Task<int> PromoteBuildAsync(Build build, Channel targetChannel, IR
                 return Constants.ErrorCode;
             }
 
+            AzureDevOpsClient azdoClient = new AzureDevOpsClient(gitExecutable: null, _options.AzureDevOpsPat, Logger, temporaryRepositoryPath: null);
```

Comment:
What scopes are required for the azdo PAT now? 

When running darc authenticate, the pop up editor says this:

`Create new Azure Dev Ops tokens at https://dev.azure.com/dnceng/_details/security/tokens (code read scope needed)` But I believe between this change, and the latest changes to darc add-build-to-channel, we need some build management scopes?

Reply:
Yeah, makes total sense. I'll look it up and include the doc update in this PR.